razor.sublime-syntax

```diff
@@ -0,0 +1,136 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: HTML Razor (C#)
+file_extensions:
+  - cshtml
```

Comment:
Do you think we should take `cshtml` out of the original `cshtml.sublime-syntax` file's extensions? Or leave it for now?

Reply:
I wasn't sure it this extension is used more commonly for Razor or not, so I opted to leave it for this PR so as not to break anything for anyone, but for my use case I would like it removed from the original file

Reply:
Long-term, I intend to peel `cshtml` out (along with basically all the file extensions in the list) into their own files. Razor seems like the generally accepted best choice for `cshtml`.